lib/rack/param_to_cookie/version.rb

```diff
@@ -1,7 +1,7 @@
 module Rack
   class ParamToCookie
     VERSION_MAJOR = 2
-    VERSION_MINOR = 0
+    VERSION_MINOR = 1
```

Comment:
I think it could be argued that this is a breaking change under [semver](http://semver.org/): if you were using it to set really big cookies, then upgrading to this version would cause it to quietly drop them. If it is, it should be a version major.